File: Ejercicios/Java/Clases/CuentaBancaria.java
public class CuentaBancaria {
    private String numero;
    private double saldo;

    public CuentaBancaria() {
        this("0000", 0);
    }

    public CuentaBancaria(String numero, double saldo) {
        this.numero = numero;
        this.saldo = saldo;
    }

    public CuentaBancaria(String numero) {
        this(numero, 0);
    }

    public CuentaBancaria(CuentaBancaria otra) {
        this(otra.numero, otra.saldo);
    }
}

File: Ejercicios/C#/Clases/CuentaBancaria.cs
public class CuentaBancaria
{
    public string Numero;
    public decimal Saldo;

    public CuentaBancaria()
    {
        Numero = "0000";
        Saldo = 0m;
    }

    public CuentaBancaria(string numero, decimal saldo)
    {
        Numero = numero;
        Saldo = saldo;
    }

    public CuentaBancaria(string numero)
    {
        Numero = numero;
        Saldo = 0m;
    }

    public CuentaBancaria(CuentaBancaria otra)
    {
        Numero = otra.Numero;
        Saldo = otra.Saldo;
    }
}
